generateBins.py

```diff
@@ -25,69 +13,53 @@
 
 
 # TODO Still need a good way of supplying these
-PFPR_FILE       = "rwa_pfpr2to10.asc"
+PFPR_FILE = "rwa_pfpr2to10.asc"
 POPULATION_FILE = "rwa_population.asc"
 
-
 # TODO Determine the bins computationally
 # Following bins are for Rwanda
 POPULATION_BINS = [2125, 5640, 8989, 12108, 15577, 20289, 27629, 49378, 95262, 286928]
```

Comment:
It seems like we are still missing the fine step of actually generating the population bin values using the GVF data.

Reply:
pop_bins is generating the population bins

Reply:
@PakhiAgarwal If that's the case then you should go ahead and complete the rest of the task.